css/style.css

```diff
@@ -85,4 +85,57 @@
         
     }
 }
-/*---------------- FIRST SECTION ENDS -|| WELCOME -----------------------*/
\ No newline at end of file
+/*---------------- FIRST SECTION ENDS -|| WELCOME -----------------------*/
+
+/* ------------------- DOWNLOAD-LAUNCH-VIEW  SECTION BEGINS---------------------------- */
+.container1{
+    height: 100%;
+    margin: 0 15% 0 15%;
+
+}
+.boxes-div {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-content: flex-start;
+    padding-top: 50px;
+}
+.boxes-div .box{
+    width: 400px;
+    height: 80px;
+    display: flex;
+    justify-content: space-around;
+    background-color: #D7EFFE;
+    padding: 20px;
+    font-size: 20px;
+    color: black;
+}
+.boxes-div .box img{
+    width: 48px;
+    height: 48px;
+}
+
+.boxes .box .text{
+    font-weight: bold;
+}
+
```

Comment:
If you are asked for responsive in desktop view and mobile view it should be responsive in all desktop view and all mbile view. use proper breakpoints. https://bulma.io/documentation/overview/responsiveness/. because your desktop view is not responsive in my system, i am having only 13 inch